bin/scripts/archive-fonts.sh

```diff
@@ -56,6 +58,20 @@ while read -r filename; do
     while IFS= read -d $'\0' -r descriptor; do
         path=${descriptor//|*/}
         file=${descriptor//*|/}
+        if [ "$file" = "README.md" ]; then
```

Comment:
Does this mean it packages a README.md only if the source dir has a README.md?

Reply:
`src/unpatched-fonts/Agave/README.md` + some info = `patched-fonts/Agave/README.md`
`patched-fonts/Agave/README.md` + some other info = `README.md` in archives

If no `src/unpatched-fonts/Agave/README.md` exists a dummy `patched-fonts/Agave/README.md` is created:

https://github.com/ryanoasis/nerd-fonts/blob/df626d3440645a1d49d82f18709e6e4814c638bc/bin/scripts/standardize-and-complete-readmes.sh#L154-L168